swift/apple/Firezone.xcodeproj/project.pbxproj

```diff
@@ -545,9 +545,10 @@
 			buildSettings = {
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = FirezoneNetworkExtension/FirezoneNetworkExtension_iOS.entitlements;
-				CODE_SIGN_IDENTITY = "Apple Distribution: Firezone, Inc. (47R2M6779T)";
-				CODE_SIGN_STYLE = Manual;
+				CODE_SIGN_IDENTITY = "$(CODE_SIGN_IDENTITY)";
+				CODE_SIGN_STYLE = "$(CODE_SIGN_STYLE)";
 				CURRENT_PROJECT_VERSION = 0;
```

Comment:
The generally accepted way to do this is: 

```
CODE_SIGN_STYLE = "$(inherited)";
CODE_SIGN_IDENTITY = "$(inherited)";
DEVELOPMENT_TEAM = "$(inherited)";
```

but I don't know of any issues with how you're doing it.